status-im/status-react
# [FIX #1668] Prevent UI keys from propagating to account model.

The ```:account-update``` event was being passed the key
```update-status?``` which has nothing to do with an account as
represented in the db. This resulted in a spec failure when this
key was detected on an attempt to save changes.

I've sanitized this event by pulling out the three account keys,
```[:name :photo-path :status]```
that are relevant to ```profile``` and ensuring that only they get
passed to ```:account-update```.

Profile updates now work as expected.

#1668 

### Summary:
- Update status fails due to spec failing the account model.
- We were leaking keys from the profile view to update-account
- I've constrained these keys.

### Steps to test:
- Open Status
- Login to a user
- Attempt to change status
- change should commit correctly

status: ready
